doc/Language/functions.rakudoc

```diff
@@ -1018,6 +1018,28 @@ dispatch, or it could be the inner routine from a C<wrap>.
 Fortunately, we have a series of re-dispatching tools that help us to make
 it easy.
 
+=head2 The Next Tools
+
+=begin table
+
+                    | Returns | Parameters | Referee
+    ================+=========+============+========
+    nextsame        | No      | Reuse      | next
+    nextwith        | No      | Specify    | next
+    callsame        | Yes     | Reuse      | next
+    callwith        | Yes     | Specify    | next
+    nextcallee      | Yes     | None       | next
+    samewith        | Yes     | Specify    | self
+    nextcallee's rv | Yes     | Specify    | preset
+
+=end table
+
+From the above we learn:
+
+=item C<with> always means that the parameters need to be specified
+=item C<same> at word end means the parameters of the referee will automatically be reused
+=item C<next> in an 8-letter word means that it won't return
```

Comment:
This is clever and true but not helpful. Can you describe this combination in a way that helps teach the user the difference from the others?

Reply:
I've rewritten some of the text.  My goal in these bullet points was to point out that the same word is used with different meanings.  While "with" always has the same meaning, "same" has two different meanings, and "next" has two different meanings.  I'm highlighting some gotchas.